osu.Game/Screens/Play/HUD/PerformancePointsCounter.cs

```diff
@@ -78,8 +80,16 @@ protected override void LoadComplete()
 
             if (scoreProcessor != null)
             {
-                scoreProcessor.NewJudgement += onJudgementChanged;
-                scoreProcessor.JudgementReverted += onJudgementChanged;
+                // Setup legacy score processor for legacy scores
```

Comment:
this comment is pointless. i'd much rather see xmldoc attached to the legacy score processor class explaining why it exists, and what it's doing differently.

remember that comments are the "why", not the "explain code below".

Reply:
I've changed the comment to the following:
// Legacy score may be necessary for correct pp calculation of legacy scores depending on the ruleset.
// If it's not necessary - method CreateLegacyScoreProcessor will return null and legacy score calculation will be skipped.